### ISSUE TITLE
Inference unable to place skeleton

### ISSUE THREAD

**Issue by Zachary-Ip** [talmolab/sleap#811]
# Bug description

Hello team, 

Thank you for this fantastic tool! It has been so far easy to use, however I am running into an issue where after training, I am unable to infer on new frames. When I run Predict -> Run inference after training I get this pop up. 
![Screenshot from 2022-06-23 12-15-32](https://user-images.githubusercontent.com/47134386/175380319-49ab4b36-b438-4385-9905-1b708d31fe02.png)
frames.

I can select any option (suggested frames, random frames, etc) and get the same pop up. 
user labled frames I get a new skeleton with 0 points 
![Screenshot from 2022-06-23 12-19-28](https://user-images.githubusercontent.com/47134386/175380609-ae876e29-2d8d-4267-ba82-3f970050cc24.png)

## Your personal set up
Im running: 
 - OS: Red Hat Linux 8.4 (Ootpa)
 - Version(s): [SLEAP v1.2.3, python 3.y.12]
 - SLEAP installation method  [ ] [Conda from package]

Here is my output from conda list:

# Name                    Version                   Build  Channel                                                                                                                                                 

_libgcc_mutex             0.1                 conda_forge    conda-forge                                                                                                                                           
_openmp_mutex             4.5                       2_gnu    conda-forge                                                                                                                                           
absl-py                   0.15.0                   pypi_0    pypi                                                                                                                                                  
aom                       3.3.0                h27087fc_1    conda-forge                                                                                                                                           
astunparse                1.6.3                    pypi_0    pypi                                                                                                                                                  
attrs                     21.2.0                   pypi_0    pypi                                                                                                                                                  
backports-zoneinfo        0.2.1                    pypi_0    pypi                                                                                                                                                  
bzip2                     1.0.8                h7f98852_4    conda-forge                                                                                                                                           
c-ares                    1.18.1               h7f98852_0    conda-forge                                                                                                                                           
ca-certificates           2022.6.15            ha878542_0    conda-forge                                                                                                                                           
cached-property           1.5.2                hd8ed1ab_1    conda-forge                                                                                                                                           
cached_property           1.5.2              pyha770c72_1    conda-forge                                                                                                                                           
cachetools                4.2.4                    pypi_0    pypi                                                                                                                                                  
cattrs                    1.1.1                    pypi_0    pypi                                                                                                                                                  
certifi                   2021.10.8                pypi_0    pypi                                                                                                                                                  
charset-normalizer        2.0.12                   pypi_0    pypi                                                                                                                                                  
clang                     5.0                      pypi_0    pypi                                                                                                                                                  
colorama                  0.4.4                    pypi_0    pypi                                                                                                                                                  
commonmark                0.9.1                    pypi_0    pypi                                                                                                                                                  
cuda-nvcc                 11.3.58              h2467b9f_0    nvidia                                                                                                                                                
cudatoolkit               11.3.1               ha36c431_9    nvidia                                                                                                                                                
cudnn                     8.2.1.32             h86fa8c9_0    conda-forge                                                                                                                                           
cycler                    0.11.0                   pypi_0    pypi                                                                                                                                                  
efficientnet              1.0.0                    pypi_0    pypi                                                                                                                                                  
ffmpeg                    5.0.1                habc3f16_3    conda-forge                                                                                                                                           
flatbuffers               1.12                     pypi_0    pypi                                                                                                                                                  
fonttools                 4.33.3                   pypi_0    pypi                                                                                                                                                  
freetype                  2.10.4               h0708190_1    conda-forge                                                                                                                                           
gast                      0.4.0                    pypi_0    pypi                                                                                                                                                  
geos                      3.9.1                h9c3ff4c_2    conda-forge                                                                                                                                           
gettext                   0.19.8.1          h73d1719_1008    conda-forge                                                                                                                                           
gmp                       6.2.1                h58526e2_0    conda-forge                                                                                                                                           
gnutls                    3.7.6                hf3e180e_5    conda-forge                                                                                                                                           
google-auth               1.35.0                   pypi_0    pypi                                                                                                                                                  
google-auth-oauthlib      0.4.6                    pypi_0    pypi                                                                                                                                                  
google-pasta              0.2.0                    pypi_0    pypi                                                                                                                                                  
grpcio                    1.44.0                   pypi_0    pypi                                                                                                                                                  
h5py                      3.1.0           nompi_py37h1e651dc_100    conda-forge                                                                                                                                    
hdf5                      1.10.6          nompi_h6a2412b_1114    conda-forge                                                                                                                                       
icu                       70.1                 h27087fc_0    conda-forge                                                                                                                                           
idna                      3.3                      pypi_0    pypi                                                                                                                                                  
image-classifiers         1.0.0                    pypi_0    pypi                                                                                                                                                  
imageio                   2.15.0                   pypi_0    pypi                                                                                                                                                  
imgaug                    0.4.0                    pypi_0    pypi                                                                                                                                                  
imgstore                  0.2.9                    pypi_0    pypi  
importlib-metadata        4.11.1                   pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h166bdaf_1    conda-forge
jsmin                     3.0.1                    pypi_0    pypi
jsonpickle                1.2                      pypi_0    pypi
keras                     2.6.0                    pypi_0    pypi
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
keyutils                  1.6.1                h166bdaf_0    conda-forge
kiwisolver                1.4.2                    pypi_0    pypi
krb5                      1.19.3               h3790be6_0    conda-forge
lame                      3.100             h7f98852_1001    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
lerc                      3.0                  h9c3ff4c_0    conda-forge
libblas                   3.9.0           15_linux64_openblas    conda-forge
libcblas                  3.9.0           15_linux64_openblas    conda-forge
libcurl                   7.83.1               h7bff187_0    conda-forge
libdeflate                1.10                 h7f98852_0    conda-forge
libdrm                    2.4.111              h166bdaf_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 12.1.0              h8d9b700_16    conda-forge
libgfortran-ng            12.1.0              h69a702a_16    conda-forge
libgfortran5              12.1.0              hdcd56e2_16    conda-forge
libgomp                   12.1.0              h8d9b700_16    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
libidn2                   2.3.2                h7f98852_0    conda-forge
liblapack                 3.9.0           15_linux64_openblas    conda-forge
libnghttp2                1.47.0               h727a467_0    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libopenblas               0.3.20          pthreads_h78a6416_0    conda-forge
libpciaccess              0.16                 h516909a_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-ng              12.1.0              ha89aaad_16    conda-forge
libtasn1                  4.18.0               h166bdaf_1    conda-forge
libtiff                   4.3.0                h0fcbabc_4    conda-forge
libunistring              0.9.10               h7f98852_0    conda-forge
libva                     2.14.0               h7f98852_0    conda-forge
libvpx                    1.11.0               h9c3ff4c_3    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.9.14               h22db469_0    conda-forge
libzlib                   1.2.12               h166bdaf_1    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
markdown                  3.3.6                    pypi_0    pypi
matplotlib                3.5.2                    pypi_0    pypi
ncurses                   6.3                  h27087fc_1    conda-forge
nettle                    3.8                  hc379101_0    conda-forge
networkx                  2.6.3                    pypi_0    pypi
numpy                     1.19.5           py37h3e96413_3    conda-forge
oauthlib                  3.2.0                    pypi_0    pypi
olefile                   0.46               pyh9f0ad1d_1    conda-forge
opencv-python             4.5.5.62                 pypi_0    pypi
opencv-python-headless    4.5.5.62                 pypi_0    pypi
openh264                  2.1.1                h780b84a_0    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openssl                   1.1.1p               h166bdaf_0    conda-forge
opt-einsum                3.3.0                    pypi_0    pypi
p11-kit                   0.24.1               hc5aa10d_0    conda-forge
packaging                 21.3                     pypi_0    pypi
pandas                    1.3.5            py37he8f5f7f_0    conda-forge
pillow                    8.4.0            py37h0f21c89_0    conda-forge
pip                       22.0.3             pyhd8ed1ab_0    conda-forge
protobuf                  3.19.4                   pypi_0    pypi
psutil                    5.9.0                    pypi_0    pypi
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pygments                  2.12.0                   pypi_0    pypi
pykalman                  0.9.5                    pypi_0    pypi
pyparsing                 3.0.7                    pypi_0    pypi
pyside2                   5.14.1                   pypi_0    pypi
python                    3.7.12          hb7a2778_100_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-rapidjson          1.6                      pypi_0    pypi
python_abi                3.7                     2_cp37m    conda-forge
pytz                      2022.1             pyhd8ed1ab_0    conda-forge
pytz-deprecation-shim     0.1.0.post0              pypi_0    pypi
pywavelets                1.3.0                    pypi_0    pypi
pyzmq                     22.3.0                   pypi_0    pypi
qimage2ndarray            1.8.3                    pypi_0    pypi
readline                  8.1.2                h0f457ee_0    conda-forge
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rich                      10.16.1                  pypi_0    pypi
scikit-image              0.19.2                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scikit-video              1.1.11                   pypi_0    pypi
scipy                     1.7.3            py37hf2a6cf1_0    conda-forge
seaborn                   0.11.2                   pypi_0    pypi
segmentation-models       1.0.1                    pypi_0    pypi
setuptools                59.8.0           py37h89c1867_1    conda-forge
setuptools-scm            6.3.2                    pypi_0    pypi
shapely                   1.7.1            py37h48c49eb_5    conda-forge
shiboken2                 5.14.1                   pypi_0    pypi
six                       1.15.0             pyh9f0ad1d_0    conda-forge
sleap                     1.2.3                    pypi_0    pypi
sqlite                    3.38.5               h4ff8645_0    conda-forge
svt-av1                   1.1.0                h27087fc_1    conda-forge
tensorboard               2.6.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow                2.6.3                    pypi_0    pypi
tensorflow-estimator      2.6.0                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2021.11.2                pypi_0    pypi
tk                        8.6.12               h27826a3_0    conda-forge
tomli                     2.0.1                    pypi_0    pypi
typing-extensions         3.10.0.2                 pypi_0    pypi
tzdata                    2022.1                   pypi_0    pypi
tzlocal                   4.2                      pypi_0    pypi
urllib3                   1.26.8                   pypi_0    pypi
werkzeug                  2.0.3                    pypi_0    pypi
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
wrapt                     1.12.1                   pypi_0    pypi
x264                      1!161.3030           h7f98852_1    conda-forge
x265                      3.5                  h924138e_3    conda-forge
xorg-fixesproto           5.0               h7f98852_1002    conda-forge
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxfixes            5.0.3             h7f98852_1004    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
zipp                      3.7.0                    pypi_0    pypi
zlib                      1.2.12               h166bdaf_1    conda-forge
zstd                      1.5.2                h8a70e8d_1    conda-forge

## How to reproduce
1. Add video, create skeleton, label frames according to (https://sleap.ai/tutorials/tutorial.html).
2. Preducit -> Run Training: Predict on -> random, epochs 200 -> 5
3. Run
4. Predict -> Run Inference -> Errpr

**Comment by roomrys** [comment]
Hi @Zachary-Ip,

This problem is usually a sign that the model parameters need to be tweaked. Do you mind sharing the parameters used in your model for training (either a screenshot of the Training Pipeline + Model Configuration GUI - with your config file selected - and/or the training_config.json file used).

Thanks,
Liezl

**Comment by roomrys** [comment]
Hi @Zachary-Ip,

Do you mind sharing some info about the model used? A screenshot of the Training Pipeline + Model Configuration GUI will be helpful. The "empty" skeleton is a result of low confidence scores of where the nodes are located... SLEAP is uncertain where any node is located in this case. I'd like to check whether changing the model parameters (and/or pipeline used) will help train a more confident model/predictor.

Thanks,
Liezl
